spec.html

```diff
@@ -24405,13 +24282,12 @@ <h1>Tail Position Calls</h1>
     <emu-clause id="sec-isintailposition" type="abstract operation">
       <h1>
         Static Semantics: IsInTailPosition (
-          _call_: unknown,
+          _call_: a |CallExpression| Parse Node, an |OptionalChain| Parse Node, or a |MemberExpression| Parse Node,
```

Comment:
I'm not convinced the extra precision over just `a Parse Node` is valuable here.

Reply:
In particular, when an AO takes exactly one kind of parse node, it's helpful to know that, but when it takes more than one it's a lot less so.